src/steps/ms-defender/findings/converters.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  createIntegrationEntity,
+  createDirectRelationship,
+  Entity,
+  Relationship,
+  RelationshipClass,
+  createMappedRelationship,
+  parseTimePropertyValue,
+} from '@jupiterone/integration-sdk-core';
+
+import { Finding } from '../../../types';
+import { Entities, MappedRelationships } from '../../../constants';
+
+export function createFindingKey(id: string): string {
+  return `${Entities.FINDING._type}:${id}`;
+}
+
+const SEVERITY_TO_NUMERIC_SEVERITY_MAP = new Map<string, number>([
+  ['low', 2],
+  ['medium', 5],
+  ['high', 7],
+  ['critical', 10],
+]);
+
+export function getNumericSeverityFromIssueSeverity(
+  issueSeverity: string,
+): number {
+  const numericSeverity = SEVERITY_TO_NUMERIC_SEVERITY_MAP.get(
+    issueSeverity.toLowerCase(),
+  );
+  return numericSeverity === undefined ? 0 : numericSeverity;
+}
+
+export function createFindingEntity(data: Finding): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: data,
+      assign: {
+        id: data.id,
+        _type: Entities.FINDING._type,
+        _class: Entities.FINDING._class,
+        _key: createFindingKey(data.id),
+        name: data.name,
+        displayName: data.name,
+        description: data.description,
+        severity: data.severity,
```

Comment:
`severity` should be Title Case. Looks like it's not based on numeric severity map above.